test/integration-tests/Seaport.test.ts

```diff
@@ -141,4 +157,187 @@ describe('Seaport', () => {
       router['execute(bytes,bytes[],uint256)'](commands, inputs, DEADLINE, { value: seaportValue })
     ).to.be.revertedWith('ExecutionFailed(0, "0x8baa579f")')
   })
+
+  describe('Seaport SELL 721', async () => {
+    let tubbyCats: ERC721
```

Comment:
opinionated nit (don't really need to address), if we did just keep using covens we wouldn't have to make more params for more nfts, might feel neater.

Reply:
Yeah, I think I'll port over to covens, I was just trying to find a quick POC txn. Would also help cut down on params for sure